lib/rester/client.rb

```diff
@@ -20,6 +20,9 @@ def initialize(adapter, params={})
       @error_threshold = (params[:error_threshold] || 3).to_i
       @retry_period = (params[:retry_period] || 1).to_f
       @logger = params[:logger] || Logger.new(STDOUT)
+      @_breaker_enabled = params[:circuit_breaker_enabled].nil? ?
+        ENV['RACK_ENV'] != 'test' : params[:circuit_breaker_enabled]
+
```

Comment:
Also, any thoughts on setting `@_request_breaker` to `proc { |*args| _request(*args) }` if the circuit breaker is not enabled?

Maybe rename it to `@_requester`.  Then set it to the circuit breaker if it's enabled, otherwise, set it to the proc above.